packages/admin-api/src/routes/notification.route.ts

```diff
@@ -0,0 +1,12 @@
+import * as express from "express"
+
+import * as notificationController from "../controllers/notification.controller"
+
+const router = express.Router()
+
+router.get("/", notificationController.getNotification)
+router.get("/check", notificationController.checkNotificationEmail)
```

Comment:
I think this should be a PUT request since it updates an existing resource and since GET requests should not have side effects.